### PR TITLE
Category navbar

### PR DESCRIPTION
**Pull request by YosefAshenafi**
# Screenshot

image

image

image

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -8,17 +8,30 @@ import Link from 'next/link';
...
-                        {category}
-                    </Link>
-                ))}
-            </div>
+            <Navbar expand="lg" className="main-navbar mobile-navbar">
+                <Container>
+                    <Navbar.Toggle aria-controls="main-navbar-nav" />
+                    <Navbar.Offcanvas id="main-navbar-nav" placement="end">
+                        <Offcanvas.Header closeButton>
+                            <Offcanvas.Title className="d-flex align-items-center"></Offcanvas.Title>
```
Comment (gwainor) [inline src/app/page.tsx]: Why is this Empty? put `Categories` inside of it

File: src/app/page.tsx [hunk 2]
```diff
@@ -8,17 +8,30 @@ import Link from 'next/link';
...
+                        </Offcanvas.Header>
+                        <Offcanvas.Body>
+                            <Nav className="justify-content-end flex-grow-1 pe-3">
+                                {data.categories.map((category) => (
+                                    <Link
+                                        className="category-labels"
+                                        key={category}
+                                        href={buildCategoryUrl(category)}
+                                    >
+                                        <Nav.Link href="/">{category}</Nav.Link>
```
Comment (gwainor) [inline src/app/page.tsx]: Why are there two links? Come on man! Pay attention!

File: src/scss/theme/_navbar.scss [hunk 3]
```diff
@@ -25,9 +25,14 @@
           animation: headerSlideDown 0.95s ease forwards;
 }
 
+.mobile-navbar {
+  float: right;
+  margin-right: -12px;
+}
```
Comment (gwainor) [inline src/scss/theme/_navbar.scss]: What is this for? Noone using `float` anymore.

File: src/scss/theme/_navbar.scss [hunk 4]
```diff
@@ -25,9 +25,14 @@
...
+
 .category-labels {
     color: $black;
     text-decoration: none;
-    margin-right: 25px;
+    margin-right: 11px;
     font-size: 12px;
 }
```
Comment (gwainor) [inline src/scss/theme/_navbar.scss]: Missing empty line on end of file. Install editorconfig vscode extension!

**Review by gwainor — CHANGES_REQUESTED** [review]

File: src/app/page.tsx (hunk 2 above)
Comment (YosefAshenafi) [reply src/app/page.tsx]: I was in a doubt of choosing among the two and left both. removing the link now.

File: src/app/page.tsx [hunk 5]
```diff
@@ -8,17 +8,30 @@ import Link from 'next/link';
...
+                                    </Link>
+                                ))}
+                            </Nav>
+                            <Nav className="d-none d-md-flex align-items-center"></Nav>
```
Comment (gwainor) [inline src/app/page.tsx]: Why did you remove the color mode toggle button?

**Review by jontsai — APPROVED** [review]
LFTM

**Review by gwainor — APPROVED** [review]
LGTM
